### PR TITLE
Add additional forbidden tables.

### LINKED ISSUE #2853: Possible Mail Lua Glitch Classic
## Describe the bug
Apparently there is a gold scamming weakaura, although I havent tried replicating this. I feel that if its something real it needs to be looked into. Granted this wasn't me that this happened to. I'm aware of how auraenviorments works.
https://www.reddit.com/r/classicwow/comments/kmw4jp/streamer_gets_scammed_for_30k_gold/?utm_medium=android_app&utm_source=share

**Are you on World of Warcraft Classic or Retail?**

- [X ] Classic
- [ ] Retail

### PR DESCRIPTION
**Pull request by nullKomplex**
# Description
Description intentionally left blank.

Fixes #2853
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested

- [x] Attempted to access forbidden tables. Error was not as expected, but blocking access worked.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings

### CODE REVIEW

File: WeakAuras/AuraEnvironment.lua [hunk 1]
```diff
@@ -173,6 +173,9 @@ local blockedFunctions = {
 
 local blockedTables = {
   SlashCmdList = true,
+  SendMailMailButton = true,
+  SendMailMoneyGold = true,
```
Comment (InfusOnWoW) [inline WeakAuras/AuraEnvironment.lua:177]: That sounds like a function

**Review by InfusOnWoW — APPROVED** [review]

File: WeakAuras/AuraEnvironment.lua (hunk 1 above)
Comment (mrbuds) [reply WeakAuras/AuraEnvironment.lua:177]: image

**Comment by perich** [comment]
Aren't there valid use cases for mailbox automation? Do I need to fork WA2 if I want to automate rebalancing on bank alts now?

**Comment by InfusOnWoW** [comment]
Mailbox automation is out of scope for WeakAuras, so yes.

**Comment by nullKomplex** [comment]
Two of the three were previously blocked API calls. These have been intended to be blocked for years. That they were still possible was an oversight.
The third I found similar enough to `MailFrame`, which was included in the original years old blocklist.

**Pull request merged**